aexpect/remote.py

```diff
@@ -918,7 +922,7 @@ def transfer(*args, **kwargs):
 @throughput_transfer
 def copy_files_to(address, client, username, password, port, local_path,
                   remote_path, limit="", log_filename=None, log_function=None,
-                  verbose=False, timeout=600, interface=None, filesize=None,
+                  verbose=False, timeout=600, interface=None, _filesize=None,
```

Comment:
Changing the argument names is tricky as people might be using them (even though they are not used later). Better solution is to either add `pylint: disable=...` or to use `del(filesize)` in the function body (it deletes it in the end anyway...)

Reply:
I still wonder what is the use of this argument anyway but I will just rely on `pyling: disable=unused-argument` for now.